### DISCUSSION TITLE
How do I escape commas in monitor config lines?

### DISCUSSION THREAD

**Discussion by Hubro** [hyprwm/Hyprland#2130] [Q&A]
I want to reuse my hyprland config across multiple machines, so I can't configure ports like DP-1 directly, I need to use monitor descriptions.

Unfortunately, one of my monitors has a comma in the description:

```
Monitor DP-2 (ID 0):
	3840x2160@60.000000 at 0x0
	description: GIGA-BYTE TECHNOLOGY CO., LTD. M28U **** (DP-2)
	make: GIGA-BYTE TECHNOLOGY CO., LTD.
	model: M28U
	serial: ****
	active workspace: 9 (9)
	reserved: 0 0 0 28
	scale: 1.50
	transform: 0
	focused: yes
	dpmsStatus: 1
	vrr: 1
```

If I try to use this description unchanged, I get a syntax error:

```
desc:GIGA-BYTE TECHNOLOGY CO., LTD. M28U 22120B003932
```

I have tried every method to escape the comma I can think of (double comma, backslash escape, deleting it altogether, quoting the entire desc with double and single quotes) but nothing works.

How do I escape this comma so I can target the correct monitor?

**Comment by vaxerski** [comment]
you cant, lol

maybe I should sanitize the description
